code/src/reducers/start.js

```diff
@@ -0,0 +1,79 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const start = createSlice({
+  name: 'start',
+  initialState: {
+    username: null,
+    description: '',
+    actions: [],
+    currentPosition: null,
+    coordinates: '',
+    loading: false,
+  },
+  reducers: {
```

Comment:
I think you could do this with a bit fewer rows. What are u actually changing in here? We only had setUsername and setCurrentPosition, and inside current position we got all alla the data (descriptopn, coordinates and actions)